runtime/src/iree/builtins/ukernel/exported_bits.h

```diff
@@ -86,13 +86,13 @@
 // query_tile_sizes
 //===----------------------------------------------------------------------===//
 
-// OPERAND_ROLE describes the role that a tensor plays in an
+// OPERAND_INDEX describes the index that a tensor plays in an
 // operation, e.g. "left-hand-size operand" (e.g. in a matmul).
-#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_ROLE_MASK 0xFF
-#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_ROLE_NONE 0x00
-#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_ROLE_LHS 0x01
-#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_ROLE_RHS 0x02
-#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_ROLE_RESULT 0x03
+#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_INDEX_MASK 0xFF
+#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_INDEX_NONE 0x00
+#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_INDEX_LHS 0x01
+#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_INDEX_RHS 0x02
+#define IREE_UK_FLAG_QUERY_TILE_SIZES_OPERAND_INDEX_RESULT 0x03
```

Comment:
Sorry, I had forgotten about that `QUERY_TILE_SIZES_` stuff. For that, revert OPERAND_INDEX to OPERAND_ROLE, since that's still old-style symbolic enums. This stuff is legacy anyway so no need to bother updating it for now.

Reply:
done.